src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,20 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(ciudad: Ciudad,private var vendedores: MutableList<Vendedor>) {
+    fun agregarVendedores(vendedor: Vendedor) {
+        if(vendedores.contains(vendedor)){
+            throw Exception("YA EXISTE VENDEDOR")
+        }
+        else{
+            vendedores.add(vendedor)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if(vendedores.contains(vendedor)){
            throw Exception("YA EXISTE VENDEDOR")
        }
        
        vendedores.add(vendedor)
```